wp-content/themes/archivepoliticalads/content-explore_candidates.php

```diff
@@ -1,34 +1,82 @@
+                        $highestCount = 0;
+                    ?>
+                    <div role="tabpanel" class="tab-pane <?php if ($i == 0) echo 'active';?>" id="<?php echo($race['name']); ?>">
+                        <ol class="explore-list">
+                            <?php
+                                $candidates = get_candidates();
+                                $first = true;
+                                $j = 0;
+                                foreach($candidates as $candidate) {
+
+
+
+                                    if ($candidate['race'] == $race['name']){
+
+                                        if ($first) {
+                                          $highestCount = $candidate['count'];
+                                          $first = false;
+                                        }
+
+                                    $j++;
+
+                                    ?>
+                                      <li class="explore-item item">
+                                        <div class="explore-label">
+                                          <p>
+                                            <?php echo($candidate['name']); ?>
+                                          </p>
+                                          <small>
+                                              <a href="<?php bloginfo('url'); ?>/browse/?q=<?php echo(urlencode("candidate:\"".$candidate['name']."\""));?>">View Ads</a>
+                                          </small>
+                                        </div>
+                                        <div class="explore-bar-container">
+                                            <div class="explore-bar" style="width:<?php echo((($candidate['count']/$highestCount)*100).'%;') ?>"></div>
+                                            <div class="explore-count"><?php echo($candidate['count']); ?>  Ad<?php echo($candidate['count']==1?"":"s");?></div>
+                                        </div>
+                                      </li>
+                                      <?php } ?>
+
+                            <?php
+
+                                  if ($j == 4){
+                                    echo '</ol><div class="collapse" id="seeMoreCandidates'.$race['name'].'"><ol class="explore-list">';
+                                  }
+
+
+
+                                }
+                            ?>
                             </div>
-                        </div>
-                    </a>
-                    <?php
-                }
-            ?>
+                        </ol>
+                        <button class="btn explore-show-more" role="button" data-toggle="collapse" data-target="#seeMoreCandidates<?php echo $race['name']; ?>" aria-expanded="false" aria-controls="seeMoreCandidates">Show / Hide <?php echo ($j-4);?> More Candidates</button>
+                    </div>
+                <?php $i++; } ?>
```

Comment:
Same thing about `$i` as `$j` above -- I think you can avoid managing it by using the $key => $val foreach structure.